Dockerfile.mysql

```diff
@@ -0,0 +1,12 @@
+FROM mysql:8.0
+
+ENV MYSQL_ROOT_PASSWORD=root_password
+ENV MYSQL_DATABASE=app_db
+ENV MYSQL_USER=app_user
+ENV MYSQL_PASSWORD=1234
+
+EXPOSE 3306
+
+VOLUME /var/lib/mysql
+
+COPY init.sql /docker-entrypoint-initdb.d/
```

Comment:
![image](https://github.com/user-attachments/assets/ab127628-def2-483c-bbf3-5071b2d45511)
Add Eof(End-of-file), need to add a blank line at the end